tests/unit/src/test/scala/tests/CompletionLspSuite.scala

```diff
@@ -229,4 +229,54 @@ class CompletionLspSuite extends BaseCompletionLspSuite("completion") {
       )
     } yield ()
   }
+
+  test("with-exclusions") {
+    cleanWorkspace()
+    for {
+      _ <- server.initialize(
+        """/metals.json
+          |{
+          |  "a": {}
+          |}
+          |/a/src/main/scala/a/A.scala
+          |package a
+          |
+          |object Main {
+          |  // @@
+          |}
+          |""".stripMargin
+      )
+      _ <- assertCompletion(
+        "Duration@@",
+        """|Duration - java.time
+           |Duration - scala.concurrent.duration
+           |DurationConversions - scala.concurrent.duration
+           |DurationDouble - scala.concurrent.duration.package
+           |DurationDouble - scala.concurrent.duration.package
+           |DurationInt - scala.concurrent.duration.package
+           |DurationInt - scala.concurrent.duration.package
+           |DurationIsOrdered - scala.concurrent.duration.Duration
+           |DurationLong - scala.concurrent.duration.package
+           |DurationLong - scala.concurrent.duration.package
+           |FiniteDuration - scala.concurrent.duration
+           |FiniteDurationIsOrdered - scala.concurrent.duration.FiniteDuration""".stripMargin,
+        includeDetail = false
+      )
+      _ <- server.didChangeConfiguration(
+        """{
+          |  "excluded-packages": [
+          |    "scala.concurrent/"
```

Comment:
user will type probably `scala.concurrent` without '/'

Reply:
Good point, I'll remove the `/`.